### DISCUSSION TITLE
folder layout question

### DISCUSSION THREAD

**Discussion by balajeek** [papermerge/papermerge-core#8] [General]
I installed papermerge docker form linuxserver.io

After setup i added 3 documents, added one folder called "Honda Financial", added one automate that tells when "Honda Financial' is found move it to "Honda Financial" folder. 

Now i am going in the media folder from mac's finder expecting to see Honda folder and the 3 files i uploaded that contains "Honda financial" content.  seem the automate worked and moved to the Honda folder.  From the web ui i see the folder i created 'Honda Financial' correctly, but on mac's finder i see the folder structure is totally different.  (see image below).  The whole point for me to try this papermerge is i want to keep the folder names like how i created it, so i could access files readonly directly from finder when i need it.  but i see all weird folders and document names.  Is this how its going to be?

![image](https://user-images.githubusercontent.com/19213088/132426251-16939a5d-5ac1-41e6-98a5-0472e5ed21c6.png)

![image](https://user-images.githubusercontent.com/19213088/132426284-abe41b9f-e315-42d2-8876-9fcd36707ead.png)

![image](https://user-images.githubusercontent.com/19213088/132426310-c7c28bc6-dd66-4d18-be9b-e0bd7e4fa4c6.png)

**Comment by balajeek** [comment]
anybody ?

**Comment by firmicus** [comment]
This is by design, unfortunatelly...
